pypeit/spectrographs/keck_mosfire.py

```diff
@@ -5,14 +5,14 @@
 import numpy as np
 from astropy.io import fits
 
-from pkg_resources import resource_filename
-
 from pypeit import msgs
 from pypeit import telescopes
 from pypeit.core import parse
 from pypeit.core import framematch
 from pypeit.par import pypeitpar
 from pypeit.spectrographs import spectrograph
+from pkg_resources import resource_filename
```

Comment:
Why was this moved? `pkg_resources` is a distribution level standard library.  It should be imported even before numpy...

Reply:
Moved it up. Not sure why it was moved.